source/administration/changelog.md

```diff
@@ -4,6 +4,66 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.13 - [Quality Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+Release Date 2019-07-16
+
+### Compatibility
+
+#### Removed and Deprecated Features
+
+We are removing support for Internet Explorer (IE11) in Mattermost v5.16.0, which releases on October 16, 2019. Learn more in our [forum post](https://forum.mattermost.org/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575).
+
```

Comment:
We could also add a reminder that end of life for 4.10 ESR is July 16th as well; new ESR is 5.9.

Reply:
Added.